packages/ERTP/src/amountMath.js

```diff
@@ -149,7 +149,7 @@ const optionalBrandCheck = (allegedBrand, brand) => {
  * @param {Brand<K> | undefined} brand
  * @returns {MathHelpers<*>}
  */
-const checkLRAndGetHelpers = (leftAmount, rightAmount, brand = undefined) => {
+const checkLRAndGetHelpers = (leftAmount, rightAmount, brand) => {
```

Comment:
I understand it to be documentation of the expected params because it has no runtime effect. For documentation of expected params, we have JSDoc so I think we should use it consistently.